### PR TITLE
raiden.cpp: Document the configuration bytes, convert comments to cpp-style

### PR DESCRIPTION
**Pull request by einstein95**
Also documented the values of 0x1fffd and 0x1fffe in the sets which didn't have them. I'm unsure whether to actually document them, as the value after decryption changes for some sets.

### CODE REVIEW

**Comment by happppp** [comment]
Line separators written as /*galaxy*/ don't need to be changed to // style.

**Comment by einstein95** [comment]
Restored the `/*galaxy*/`s

File: src/mame/seibu/raiden.cpp [hunk 1]
```diff
@@ -334,25 +348,25 @@ WRITE_LINE_MEMBER(raiden_state::vblank_irq)
...
 	config.set_maximum_quantum(attotime::from_hz(12000));
 
-	/* video hardware */
+	// video hardware
 	BUFFERED_SPRITERAM16(config, m_spriteram);
 
 	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
 	screen.set_refresh_hz(59.60); // verified on pcb */
-	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500) /* not accurate */);
+	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500) // not accurate);
```
Comment (0kmg) [inline src/mame/seibu/raiden.cpp:369]: You've commented out the closing parenthesis and semicolon. What was the comment doing *inside* the argument list anyway? I suspect a vast network of saboteurs conspiring to undermine the Seibu drivers. This message will self-destruct in...5...4...

Comment (einstein95) [reply src/mame/seibu/raiden.cpp]: That is really weird. Fixed it anyway

File: src/mame/seibu/raiden.cpp [hunk 2]
```diff
@@ -6,18 +6,22 @@
...
 
-    High nibble: Player respawn behavior when single playing
-        0x0*  = If single playing, Restart at checkpoint when every miss
-        0x8*  = Respawn instantly when every miss
+    The byte at 0xffffd (0x1fffe in program rom 4) in the main cpu region controls the
+    country/game mode:
+    High nibble: Player respawn behavior in single player
+        0x0*  = Restart at checkpoint
+        0x8*  = Respawn instantly
+        0xA*  = Unknown, identical to 0x0*? (seen in raidenk)
```
Comment (cuavas) [inline src/mame/seibu/raiden.cpp:24]: Does it actually check for specific values, or is it just that bit 0 of 0xffffb controls the FBI screen and bit 7 of 0xffffd controls respawn behaviour?

Comment (einstein95) [reply src/mame/seibu/raiden.cpp:24]: The lower nibble of 0xffffb is explicitly checked against 1h (code at `ef299` in raiden)
It appears that bit 7 of 0xffffd is checked, I've updated that part of the comment

**Pull request merged**